Components/Core/Timer.cpp

```diff
@@ -12,6 +12,11 @@
 */
 void empty_callback(TimerHandle_t rtTimerHandle) {};
 
+//CurrentState Timer::completed()
+//{
+//	timerState = COMPLETE;
+//}
+
```

Comment:
Don't keep unused code, unless you plan to use it

Reply:
was not going to use that code so I removed it